package.json

```diff
@@ -1,6 +1,6 @@
 {
   "devDependencies": {
-    "@salesforce/dev-config": "1.1.0",
+    "@salesforce/dev-config": "1.4.1",
```

Comment:
Updating @salesforce/dev-config, since one of its dependencies has a peer dependency on typescript